src/biguint.rs

```diff
@@ -1912,30 +2078,66 @@ pub fn to_str_radix_reversed(u: &BigUint, radix: u32) -> Vec<u8> {
     res
 }
 
+/// Creates and initializes a `BigUint`.
+///
+/// The digits are in little-endian base matching `BigDigit`.
+///
+/// This is an internal `pub(crate)`-ish API only!
+#[inline]
+pub fn biguint_from_vec(digits: Vec<BigDigit>) -> BigUint {
```

Comment:
This function is `pub`, but it requires a `Vec<BigDigit>`, so it effectively can't be used externally. The `#[doc(hidden)]` attribute would hide it form public documentation. There technically could be a name conflict if someone did `use num_bigint::*`, but that seems unlikely.

Reply:
This is in the private `mod biguint` though, so it's truly inaccessible outside of the crate, as long as we don't accidentally re-export it elsewhere. It should be made `pub(crate)` whenever we raise our rustc minimum sufficiently though.

Reply:
It seems like half my comments are due to oversight on my part :/